### PR TITLE
Adding info labels to EditSessionView

### PR DESCRIPTION
**Pull request by SzymonGorowski**
https://trello.com/c/vFLVu4mA/713-adding-labels-to-editsessionview

### CODE REVIEW

File: AirCasting/SessionViews/EditView.swift [hunk 1]
```diff
@@ -77,4 +82,10 @@ struct EditView<VM: EditViewModel>: View {
             presentationMode.wrappedValue.dismiss()
         }.buttonStyle(BlueTextButtonStyle())
     }
+    
+    private var customSpacer: some View {
+        Rectangle()
+            .fill(Color(UIColor.systemBackground))
+            .frame(height: 10)
+    }
```
Comment (jankrzempek) [inline AirCasting/SessionViews/EditView.swift:90]: Why not just a padding instead?

Comment (SzymonGorowski) [reply AirCasting/SessionViews/EditView.swift:90]: This is what happen when I added .`padding()`

Screenshot 2022-06-28 at 12 57 15

Comment (jankrzempek) [reply AirCasting/SessionViews/EditView.swift:90]: 😂 I was going to say nothing

Comment (anna1901) [reply AirCasting/SessionViews/EditView.swift:90]: @SzymonGorowski padding or spacer? 🤔

Comment (SzymonGorowski) [reply AirCasting/SessionViews/EditView.swift]: @jankrzempek & @anna1901 thank you for digging 🤦  I realized that I put the `.padding()` in the wrong place.

**Review by anna1901 — CHANGES_REQUESTED** [review]
Please change the fonts size according to designs. For labels we have to guess what would be good, but maybe there are some labels in the designs that can serve as an inspiration.

If you group labels with textFields and error messages to separate variables and put the in a VStack with some spacing (like this: `VStack(spacing: 20)`), wouldn't it remove the necessity for creating this custom spacer?

**Review by anna1901 — APPROVED** [review]

**Pull request merged**